inc/osvr/Client/ViewerEye.h

```diff
@@ -109,15 +110,23 @@ namespace client {
         getProjection(double near, double far,
                       OSVR_MatrixConventions flags) const;
 
+        /// @brief Gets clipping plane for a given eye
+        OSVR_CLIENT_EXPORT util::Rectd getRect(double near, double far) const;
```

Comment:
Can leave off the two parameters and just pass `1.0` in the implementation for near, to get the clipping planes (plural) for the given *surface*.